main.go

```diff
@@ -814,6 +858,12 @@ func parseFlags() (config, error) {
 		"The name of the HTTP header containing the tenant ID to forward to the metrics upstreams.")
 	flag.StringVar(&cfg.metrics.tenantLabel, "metrics.tenant-label", "tenant_id",
 		"The name of the PromQL label that should hold the tenant ID in metrics upstreams.")
+	flag.StringVar(&rawTracesWriteEndpoint, "traces.write.endpoint", "",
+		"The endpoint against which to make gRPC write requests for traces.")
+	flag.StringVar(&cfg.traces.upstreamCAFile, "traces.tls.ca-file", "",
+		"File containing the TLS CA against which to upstream OTLP trace servers. Leave blank to disable TLS.")
+	flag.StringVar(&cfg.traces.tenantHeader, "traces.tenant-header", "X-Tenant",
+		"The name of the HTTP header containing the tenant ID to forward to the logs upstream.")
```

Comment:
Seems like also need to regenerate README to add these options, it seem to be causing CI to fail currently.